sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 ### Added
 - Added a retryPolicy() chain method to the `CommunicationIdentityClientBuilder`
 
+### Breaking
+- `CommunicationIdentityClient.createUserWithToken` and `CommunicationIdentityAsyncClient.createUserWithToken` functions have been renamed to
+`CommunicationIdentityClient.createUserAndToken` and `CommunicationIdentityAsyncClient.createUserAndToken`
+- `CommunicationIdentityClient.createUserWithTokenWithResponse` and `CommunicationIdentityAsyncClient.createUserWithTokenWithResponse` functions have been renamed to
```

Comment:
do we normally say function or method in java?

Reply:
Python is starting to break me 💯

Reply:
Hmm actually, the examples below don't specify "functions"/"methods". 

e.g.: `CommunicationIdentityAsyncClient.issueToken` and `CommunicationIdentityClient.issueToken` is renamed to `CommunicationIdentityAsyncClient.getToken` and `CommunicationIdentityClient.getToken`.